pom.xml

```diff
@@ -1725,6 +1725,7 @@
                             <exclude>**/**/operation_logs/**/**</exclude>
                             <exclude>**/*.output.schema</exclude>
                             <exclude>**/apache-kyuubi-*-bin*/**</exclude>
+                            <exclude>**/META-INF/services/*</exclude>
```

Comment:
We can add the license to the service file, refer to

https://github.com/apache/flink/blob/master/flink-connectors/flink-connector-kafka/src/main/resources/META-INF/services/org.apache.flink.table.factories.Factory